core-web/libs/portlets/edit-ema/portlet/src/lib/edit-ema-editor/components/ema-page-dropzone/pipes/error/dot-error.pipe.ts

```diff
@@ -47,14 +50,14 @@ export class DotErrorPipe implements PipeTransform {
         };
     }
 
-    private isValidContentType(acceptTypes: string, item: EmaDragItem): boolean {
-        if (item.baseType === DotCMSBaseTypesContentTypes.WIDGET) {
+    private isValidContentType(acceptTypes: string, item: EmaDragItem | undefined): boolean {
```

Comment:
more like?
```
private isValidContentType(acceptTypes: string, item: EmaDragItem | undefined): boolean {
  if (!item) {
    return false;
  }

  if (item.baseType === DotCMSBaseTypesContentTypes.WIDGET) {
    return true;
  }

  const acceptTypesArr = acceptTypes.split(',');
  return acceptTypesArr.includes(item.contentType);
}
```
I don't like the ? in the object, I feel like I am not sure of the type.